packages/renovate-config/README.md

```diff
@@ -0,0 +1,110 @@
+# `@lmc-eu/renovate-config`
+
+> LMC's config for renovate
+
+## Usage
+
+Enable Renovate in your repo and just `extends` in `renovate.json`.
+
+```json
+{
+  "extends": ["@lmc-eu"]
+}
+```
+
+Note: You don't have to do `npm i -D @lmc-eu/renovate-config`.
+
+## Presets
+
+### `@lmc-eu`
+
+Default preset
+
+```json
+{
+  "extends": [
+    ":semanticPrefixFixDepsChoreOthers",
+    ":ignoreModulesAndTests",
+    ":autodetectPinVersions",
+    "workarounds:all",
+    ":prConcurrentLimit10"
+  ],
+  "labels": ["dependencies"],
+  "branchPrefix": "dependencies/",
+  "commitMessagePrefix": "Deps: "
+}
+```
+
+### `@lmc-eu:scheduleWeeklyNonOfficeHours`
+
+Schedule weekly non-office hours
+
+```json
+{
+  "schedule": ["after 10pm and before 7am on Monday"]
+}
+```
+
+### `@lmc-eu:groupMajorProdDependencies`
+
+Group together major prod dependencies
+
+```json
+{
+  "packageRules": [
+    {
+      "groupName": "major prod dependencies",
+      "groupSlug": "major-prod",
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["major"],
+      "matchDepTypes": ["dependencies"]
+    }
+  ]
+}
+```
+
+### `@lmc-eu:groupNonMajorProdDependencies`
+
+Group together non-major prod dependencies
+
+```json
+{
+  "packageRules": [
+    {
+      "groupName": "non-major prod dependencies",
+      "groupSlug": "non-major-prod",
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "matchDepTypes": ["dependencies"]
+    }
+  ]
+}
+```
+
+### `@lmc-eu:groupAllDevDependencies`
+
+Group together all dev dependencies
+
+```json
+{
+  "packageRules": [
+    {
+      "groupName": "all dev dependencies",
+      "groupSlug": "all-dev",
+      "separateMajorMinor": false,
+      "matchPackagePatterns": ["*"],
+      "matchDepTypes": ["devDependencies"]
+    }
+  ]
+}
+```
+
+## References
+
+- [Renovate Docs](https://renovatebot.com/docs/)
+- [Configuration Options \| Renovate Docs](https://renovatebot.com/docs/configuration-options/)
+- [Default Presets \| Renovate Docs](https://renovatebot.com/docs/presets-default/)
+
+## License
+
+[MIT License](https://opensource.org/licenses/MIT)
```

Comment:
Does BSD allow MIT for a nested package…?

Reply:
Well, do not know exactly, but both licenses are [very similar](https://fossa.com/blog/open-source-software-licenses-101-bsd-3-clause-license/).

What I want to say with this is that there is no bound to strv in this package so we can use MIT license. Although we can stay with BSD 🤷‍♂️